data.json

```diff
@@ -125,15 +125,6 @@
     "dateAdded": "2019-10-23T00:39:55.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
-  {
-    "name": "Deque",
-    "difficulty": "easy",
-    "description": "Answer Deque's feedback survey once you're logged in to get an awesome t-shirt.",
-    "reference": "https://twitter.com/dequesystems/status/1182651282869362694",
-    "image": "https://i.imgur.com/t82DlcW.jpg",
-    "dateAdded": "2020-02-25T14:33:22.000Z",
-    "tags": ["clothing"]
```

Comment:
We normally add the `expired` tag rather than removing an opportunity so people can still reference it if they so choose

// @aslafy-z please correct me if it's different when the reference disappears

Reply:
I guess we can keep it. Expired opportunities are skipped in the ci link checking step right?

Reply:
I don't think so 😬

Reply:
If you wish to see the opportunity for future it's ok, but the it won't be visible anywhere in the Deque social media. If so, you can close this PR